kubernetes-client/src/main/java/io/fabric8/kubernetes/client/DefaultKubernetesClient.java

```diff
@@ -317,6 +281,23 @@ public RootPaths rootPaths() {
   }
 
   @Override
+  public String getPodLog(String namespace, String podName, String containerName, boolean pretty) {
+    String url = getMasterUrl() + "namespaces/" + namespace + "/pods/" + podName + "/log?pretty=" + pretty;
```

Comment:
TBH this is a single function with a single result with 4 parameters; not sure how valuable it is to be in the pods DSL but if you wanna refactor stuff so it can be there its not a huge biggie either way